packages/lesmis/src/api/queries/useMapOverlayReportData.js

```diff
@@ -154,15 +155,15 @@ export const useMapOverlayReportData = ({ entityCode, year }) => {
   };
 
   const { data: measureDataResponse, isLoading: measureDataLoading } = useQuery(
-    ['mapOverlay', entityCode, selectedOverlay, params],
+    ['mapOverlay', COUNTRY_CODE, selectedOverlay, params],
     () =>
-      get(`report/${entityCode}/${reportCode}`, {
+      get(`report/${COUNTRY_CODE}/${reportCode}`, {
```

Comment:
Funny thing is, this probably results in _less_ overall data being fetched, because you often view a map overlay at the country level first before zooming in